invenio_app_ils/config.py

```diff
@@ -1365,6 +1365,12 @@ def _(x):
 
 # List of available vocabularies
 ILS_VOCABULARIES = [
+    "acq_currency",
```

Comment:
I would go for a voc. with global currencies instead, they are an ISO standard I would say...